julia/test/unittest/ndarray.jl

```diff
@@ -1525,8 +1525,8 @@ function test_argmax()
          NaN 2 6]
     x = NDArray(A)
 
-    @test copy(argmax(x, dims = 1)) == [1 1 2]
-    @test copy(argmax(x, dims = 2)) == reshape([2, 3], :, 1)
+    @test copy(argmax(x, dims = 1)) == [2 1 2]
```

Comment:
The result is consistent with Julia built-in function `argmin` and `argmax`

Reply:
Please consider update the docstring of `argmax`/`argmin` in `julia/src/ndarray/reduction.jl` as well.

Reply:
I have updated it : )